_posts/2016-02-04-ombulabs-products.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "New OmbuLabs Products"
+date: 2016-02-04 18:35:00
+categories: ["Design"]
+author: "cecilia"
+---
+
+As part of Christmas presents for the Ombu Labs' team, we worked on a series of products. We want to share with you the process and its final result.
```

Comment:
@cecilia-ombulabs Try rephrasing the first sentence to something like this: "We recently worked on a series of Christmas presents for the team. We are happy to share part of the process and its final result!"